components/users.js

```diff
@@ -149,9 +149,50 @@ SteamCommunity.prototype.postUserComment = function(userID, message, callback) {
 		}
 
 		if(body.success) {
+			const $ = Cheerio.load(body.comments_html);
+			const commentID = $('.commentthread_comment').attr('id').split('_')[1];
+
+			callback(null, commentID);
+		} else if(body.error) {
+			callback(new Error(body.error));
+		} else {
+			callback(new Error("Unknown error"));
+		}
+	}, "steamcommunity");
+};
+
+SteamCommunity.prototype.deleteUserComment = function(userID, commentID, callback) {
+	if(typeof userID === 'string') {
+		userID = new SteamID(userID);
+	}
+
+	var self = this;
+	this.httpRequestPost({
+		"uri": "https://steamcommunity.com/comment/Profile/delete/" + userID.toString() + "/-1",
+		"form": {
+			"gidcomment": commentID,
+			"start": 0,
+			"count": 1,
```

Comment:
Default is 6, but 1 will request less data from Valve. Hence, it's more efficient.
Sadly, it must return at least 1 comment html, 0 is not possible (to my knowledge). I tried `0` and `-1`.